### PR TITLE
gpui: Add example for `uniform_list`

### PR DESCRIPTION
**Pull request by qinyuhang**
GPUI: Add example for `uniform_list`

- Added example for `uniform_list`
  - Run `cargo run --example uniform_list`  to fire up the example
  - https://github.com/user-attachments/assets/bb554fbc-c097-4ce5-8077-782dc4c5c398

Release Notes:

- N/A

https://github.com/user-attachments/assets/bb554fbc-c097-4ce5-8077-782dc4c5c398

### CODE REVIEW

**Comment by qinyuhang** [comment]
@cla-bot check

File: crates/gpui/Cargo.toml [hunk 1]
```diff
@@ -192,3 +192,7 @@ path = "examples/text_wrapper.rs"
 [[example]]
 name = "opacity"
 path = "examples/opacity.rs"
+
+[[example]]
+name = "simple_list"
+path = "examples/simple_list.rs"
```
Comment (maxdeviant) [inline crates/gpui/Cargo.toml:198]:
```suggestion
path = "examples/simple_list.rs"

```

Comment (qinyuhang) [reply crates/gpui/Cargo.toml]: Hi, @maxdeviant , I've added a new line, cheers.

**Review by maxdeviant — APPROVED** [review]
Thank you!

**Pull request merged**